docs/pages/en-us/Architecture.md

```diff
@@ -125,3 +126,26 @@ This table stores information on the jobs that are requested. Entries are added
 The task tables have the same structure as the job table. Each job has one task table, which is created when the first subtask is reported, and deleted when the job completes successfully. If the job fails, the table is retained for a period of time for examination.
 
 When a task is marked complete, the system checks whether the parent task (or the job if it is the top level) can also be marked complete.
+
+### Job Dependency Table
+
+This table stores Ids of dependent jobs i.e. jobs which must be completed before the job in question can be executed.
+
+| **Column**        | **Data Type** | **Nullable?** | **Primary Key?** |
+|-------------------|---------------|---------------|------------------|
+| Job_Id            | String        | No            | Yes              |
```

Comment:
It doesn't make sense for Job_Id to be the primary key of this table.

Reply:
Primary Key should be `Yes` for both columns